koku/reporting/provider/aws/models.py

```diff
@@ -326,6 +326,7 @@ class Meta:
     unblended_cost = models.DecimalField(max_digits=17, decimal_places=9,
                                          null=True)
     resource_count = models.IntegerField(null=True)
+    tags = JSONField(null=True)
```

Comment:
I think now is also a good time to put an index on tags. If you look at `OCPUsageLineItemDailySummary` in the OCP models you can see that I used a GinIndex specific to Postgres JSON.